### PR TITLE
problem solved on Feb.11

### PR DESCRIPTION
**Pull request by ryoosukesaito**
there is a problem on leetcode which was solved on Feb.11

### CODE REVIEW

File: 2441.largest-positive-integer-that-exists-with-its-negative.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+ */
+
+/**
+ * @param {number[]} nums
+ * @return {number}
+ */
+
+var findMaxK = function (nums) {
+  let result = -1;
+  const arrNums = nums;
```
Comment (yuya-minamide) [inline 2441.largest-positive-integer-that-exists-with-its-negative.js:12]: Is there any reason to change the name of the argument here?

File: 2441.largest-positive-integer-that-exists-with-its-negative.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  for (let i = 0; i < arrNums.length; i++) {
+    for (let j = i + 1; j <= arrNums.length; j++) {
+      if (arrNums[i] + arrNums[j] === 0) {
+        let largestNum = Math.max(arrNums[i], arrNums[j]);
+        if (largestNum > result) {
+          result = largestNum;
+        }
```
Comment (yuya-minamide) [inline 2441.largest-positive-integer-that-exists-with-its-negative.js:19]:
```suggestion
       if (largestNum > result)　 result = largestNum;
``` 
You can also write like this.

File: 628.max-product-of-three-numbers.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+/**
+ * @param {number[]} nums
+ * @return {number}
+ */
+
+var maximumProduct = function (nums) {
+  nums.sort((a, b) => b - a);
+  const positiveSum = nums[0] * nums[1] * nums[2];
+  const negativeTwo = nums[0] * nums[nums.length - 1] * nums[nums.length - 2];
```
Comment (yuya-minamide) [inline 628.max-product-of-three-numbers.js:13]: I don't think these names are appropriate

File: 628.max-product-of-three-numbers.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+  if (positiveSum > negativeTwo) {
+    return positiveSum;
+  } else {
+    return negativeTwo;
```
Comment (yuya-minamide) [inline 628.max-product-of-three-numbers.js:18]:
```suggestion
  positiveSum > negativeTwo ? return positiveSum : return negativeTwo;
```
You can also write like this.

**Review by yuya-minamide — APPROVED** [review]

File: 2441.largest-positive-integer-that-exists-with-its-negative.js (hunk 2 above)
Comment (ryoosukesaito) [reply 2441.largest-positive-integer-that-exists-with-its-negative.js:19]: is that shorthand right?
im gonna fix that
thank you

**Pull request merged**